tests/init.go

```diff
@@ -246,7 +246,7 @@ var Forks = map[string]*params.ChainConfig{
 		ArrowGlacierBlock:   big.NewInt(0),
 		GrayGlacierBlock:    big.NewInt(0),
 	},
-	"Merge": {
+	"Paris": {
```

Comment:
If this causes a problem, we can just add `Merge` too, so we have them both

Reply:
I have told testing guys to unify the test data. I am fine with defining both `Merge` and `Paris` to pass the tests.

Reply:
Pushed